playbooks/roles/registry-server-setup/tasks/apply-certs.yml

```diff
@@ -4,6 +4,7 @@
   copy:
     src: "{{ suse_registry_certs_dir }}/domain.crt"
     dest: "/etc/pki/trust/anchors"
+  delegate_to: "{{ groups['soc-deployer'][0] }}"
```

Comment:
isn't the generation of the cert on the localhost anymore?

Reply:
no, the cert generation happens in the registry-server-setup role which runs on the soc-deployer hostgroup

Reply:
mmm. https://github.com/SUSE-Cloud/socok8s/blob/master/playbooks/generic-generate_certs_for_local_registry.yml that was what to reuse. Not sure how we managed to make that on a different node.